lib/chat/getChatContext.tsx

```diff
@@ -10,25 +9,15 @@ import getUsersScore from "../stack/getUsersScore";
 import { SCORE_EVENT } from "@/types/score";
 import getRecentScore from "./getRecentScore";
 import getScoresInPast24 from "./getScoresInPast24";
-import getFans from "./getFans";
-import { INSTRUCTION, NOTES } from "./const";
+import { INSTRUCTION } from "./const";
 
 const getChatContext = async () => {
   const context = [];
   const client = getSupabaseServerAdminClient();
   const scores: SCORE_EVENT[] = await getUsersScore();
 
-  context.push(NOTES);
   context.push(INSTRUCTION);
 
-  context.push(`\n\n1. Fans' Information:\t`);
-  const fanContext = await getFans(client);
-
-  context.push(fanContext);
-
-  const follows = await getFollows(client);
-  context.push(`\n2. Followers: ${follows}`);
```

Comment:
To confirm, `fan` context is being removed from this, general, chat response and migrated to the specific function call response, right?